### ISSUE TITLE
Error message "We couldn't make changes to your system. Please install or uninstall 'ipfs-npm' package manually."

### ISSUE THREAD

**Issue by khinsen** [ipfs-shipyard/ipfs-desktop#1124]
After updating to IPFS Desktop 0.9.2 (under macOS 10.14.6), every time I open "settings" I get the error message "We couldn't make changes to your system. Please install or uninstall 'ipfs-npm' package manually." But I have no idea how I would install or uninstall "ipfs-npm" manually, nor do I see how to figure out if it needs to be installed or uninstalled.

After some searching, I now understand that this is related to a new experimental feature called "npm on IPFS", but that doesn't help me with this error message. In case it helps, I have never used NPM and it is probably not installed on my system (unless something else installed it indirectly).

**Comment by hacdias** [comment]
Hey @khinsen! Thanks for reporting that back! I've never seen that message appear unless I tried to activate it and it fails... Could you please show me the contents of `~/Library/Application Support/IPFS Desktop/config.json`, the `experiments` section, please?

**Comment by khinsen** [comment]
@hacdias Here is the complete file (very short), the `experiments` section being empty:

```
{
	"ipfsConfig": {
		"type": "go",
		"path": "/Users/hinsen/.ipfs",
		"flags": [
			"--migrate=true",
			"--routing=dhtclient",
			"--enable-gc=true"
		],
		"keysize": 2048
	},
	"version": 5,
	"language": "en-GB",
	"window": {
		"width": 1241,
		"height": 777
	},
	"checkedCorsConfig": true,
	"ipfsOnPath": false,
	"experiments": {}
}
```

**Comment by hacdias** [comment]
Weird... could you try the latest version, please?

**Comment by hacdias** [comment]
This was fixed somewhere between 0.9.2 and the latest version. Closing!

**Comment by khinsen** [comment]
Updated to 0.10.1 - the problem is gone indeed. Thanks!
